oop/homework.py

```diff
@@ -77,6 +108,18 @@ class Cheetah:
       if age grosser 15(not including) return 40
 
     """
+    food = {"gazelle": 30, "rabbit": 15, "another_food": 0}
+
+    def eat(self, product):
+        self.saturation_level += self.food.get(product, 0)
+
+    def _set_average_speed(self):
+        value_average_speed = {
+            self.age <= 5: 90,
+            5 <= self.age <= 15: 75,
```

Comment:
одне значення мусить ввійти в якийсь один діапазон
```suggestion
            5 < self.age <= 15: 75,
```

Reply:
точноо) поправив) дякую

Reply:
якщо запускати кожен юніт тест окремо, то тест не падає. Падає тоді, коли запускаєш автотест всі зразу тести. Але вже поправив)

Reply:
напевно там не параметризовані тести)

Reply:
чи шось таке)

Reply:
але хороший код має працювати не тільки для того, щоб працювали тести)